cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -207,24 +209,27 @@ func executeExchangeWrapperTests(ctx context.Context, t *testing.T, exch exchang
 			e = time.Now()
 			s = e.Add(-time.Minute * 3)
 		}
-		for y := 0; y <= assetLen; y++ {
-			inputs := make([]reflect.Value, method.Type().NumIn())
-			argGenerator := &MethodArgumentGenerator{
-				Exchange:    exch,
-				AssetParams: assetParams[y],
-				MethodName:  methodName,
-				Start:       s,
-				End:         e,
-			}
-			for z := range method.Type().NumIn() {
-				argGenerator.MethodInputType = method.Type().In(z)
-				generatedArg := generateMethodArg(ctx, t, argGenerator)
-				inputs[z] = *generatedArg
-			}
-			assetY := assetParams[y].Asset.String()
-			pairY := assetParams[y].Pair.String()
+		for y := range assetLen {
+			ap := assetParams[y]
+			assetY := ap.Asset.String()
+			pairY := ap.Pair.String()
 			t.Run(methodName+"-"+assetY+"-"+pairY, func(t *testing.T) {
```

Comment:
You could replace `assetY` and `pairY` in the `t.Run()` right?

Reply:
I think he means:

```suggestion
			t.Run(methodName+"-"+ap.Asset.String()+"-"+ap.Pair.String(), func(t *testing.T) {
```

And rm the variables, but could be wrong.

Reply:
urgh. That was obvious 🤦